pkg/proxy/iptables/proxier.go

```diff
@@ -1515,7 +1515,13 @@ func (proxier *Proxier) syncProxyRules() {
 	klog.V(5).InfoS("Restoring iptables", "rules", proxier.iptablesData.Bytes())
 	err = proxier.iptables.RestoreAll(proxier.iptablesData.Bytes(), utiliptables.NoFlushTables, utiliptables.RestoreCounters)
 	if err != nil {
-		klog.ErrorS(err, "Failed to execute iptables-restore")
+		parseError, pErr := utiliptables.ParseErrorByRestore(err)
+		if pErr != nil {
+			klog.ErrorS(err, "Failed to execute iptables-restore")
+		} else {
+			lines := utiliptables.ExtractLines(3, parseError.Line(), proxier.iptablesData.Bytes())
+			klog.Error("Failed to execute iptables-restore: ", parseError.Error(), lines)
```

Comment:
```suggestion
			klog.ErrorS(parseError.Error(), "Failed to execute iptables-restore.",  "lines", lines)
```

/home/prow/go/src/k8s.io/kubernetes/pkg/proxy/iptables/proxier.go:1523:4: unstructured logging function "Error" should not be used

Reply:
maybe we should use klog.Errorln, I want the log to be displayed in a format like this：
```
 Failed to execute iptables-restore: input error on line 1234:
    1231: ...
    1232: ...
    1233: ...
    1234: ...
    1235: ...
    1236: ...
    1237: ...
```

Reply:
I'd rather build a `[]string` and log that, one line per array member